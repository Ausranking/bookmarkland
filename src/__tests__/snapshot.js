//optional snapshot

// import { render } from "@testing-library/react";
// import Counter from "../components/Counter";

// it('renders page unchanged', ()=> {
//     const {container} = render(<Counter/>)
//     expect (container).toMatchSnapshot()
// })