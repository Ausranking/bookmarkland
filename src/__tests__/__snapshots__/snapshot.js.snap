// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page unchanged 1`] = `
<div>
  <section
    class="w-full "
  >
    <div
      class=" mt-10 flex flex-col gap-5 place-items-center align-body"
    >
      <h1
        data-testid="count"
      >
        Counter: 
        0
      </h1>
      <button
        class="border bg-soft-blue p-2 rounded hover:bg-soft-blue/50"
        data-testid="incrementbtn"
      >
        Increment
      </button>
      <button
        class="border bg-soft-red p-2 rounded hover:bg-soft-red/50"
        data-testid="decrementbtn"
      >
        Increment
      </button>
    </div>
  </section>
</div>
`;
